include/robot_localization/navsat_transform.h

```diff
@@ -52,6 +52,9 @@
 
 #include <string>
 
+#include <GeographicLib/Geocentric.hpp>
```

Comment:
Nit: can we drop these headers before `<string>`? I like to keep the STL headers last.

Reply:
Done.